types/koa-helmet/v5/index.d.ts

```diff
@@ -0,0 +1,116 @@
+// Type definitions for koa-helmet 5.2
+// Project: https://github.com/venables/koa-helmet#readme
+// Definitions by: Nick Simmons <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/koa-helmet/v5/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me